src/seedu/addressbook/commands/order/DraftOrderEditPointsCommand.java

```diff
@@ -33,14 +32,20 @@ public DraftOrderEditPointsCommand(int points) {
     @Override
     public CommandResult execute() {
         try {
-            final ReadOnlyMember member = rms.getMemberFromDraftOrder();
```

Comment:
getMemberFromDraftOrder() does the same thing as draftOrder.getCustomer(). Why change it when it can be abstracted into a method?

Reply:
The reason is that the class structural feels more natural for me.
In our implementation, the draft order is an element of RMS, and the member field of the draft is an element of the draft order.
By splitting it into two steps, the shortcut method is sacrificed for codes that follow more closely to the structural design of the program.